webapp/apps/btax/models.py

```diff
@@ -0,0 +1,142 @@
+import re
+
+from django.db import models
+from django.core import validators
+from django.core.urlresolvers import reverse
+from django.core.exceptions import ValidationError
+from django.core.validators import MinValueValidator, MaxValueValidator, RegexValidator
+from django.contrib.auth.models import User
+
+from uuidfield import UUIDField
+from jsonfield import JSONField
+import datetime
+
+from ..taxbrain.models import (SeparatedValuesField as _SeparatedValuesField,
+                               CommaSeparatedField as _CommaSeparatedField,
+                               convert_to_floats)
+
+class SeparatedValuesField(_SeparatedValuesField):
```

Comment:
I'm confused regarding `SeparatedValuesField` and `CommaSeparatedField`. Why not just import them and inherit directly?